src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java

```diff
@@ -301,26 +378,68 @@ public int getIdleMinutes() {
         return (idleMinutes != null) ? idleMinutes : 0;
```

Comment:
For these cases I originally had them this way, but for testing it seems that if the config doesn't define them then the tests will error which makes me think the setter is not called for default values if the value doesn't exist in config. This is not a problem for UI users since the jelly has defined defaults. It also was not a problem for certain getters where the default java initialization is the same as the default value.